justfile

```diff
@@ -76,9 +76,16 @@ qemu: build-linux nixos-image
     -m 512M \
     -cpu host \
     -virtfs local,path={{invocation_directory()}}/..,security_model=none,mount_tag=home \
+    -virtfs local,path=/scratch,security_model=none,mount_tag=scratch \
```

Comment:
Is path= here for the VM or the host path?

Reply:
@Mic92 path is the host path. I need `/scratch` because it is where my linux kernel is (which is used for `./qemu_nested.sh`).

Reply:
Mhm. Could it be `-virtfs local,path={{invocation_directory()}}/..,security_model=none,mount_tag=scratch \`
instead? This would break on other setups where there is no `/scratch`.